db/data/20230628132154_setup_july_v2_experiments_bangladesh.rb

```diff
@@ -0,0 +1,85 @@
+
+        single_group = experiment.treatment_groups.create!(description: "single_notification")
+        single_group.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 3)
+
+        cascade1 = experiment.treatment_groups.create!(description: "cascade1")
+        cascade1.reminder_templates.create!(message: "notifications.set01.official_short", remind_on_in_days: -1)
+        cascade1.reminder_templates.create!(message: "notifications.set02.official_short", remind_on_in_days: 0)
+        cascade1.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 3)
+
+        cascade2 = experiment.treatment_groups.create!(description: "cascade2")
+        cascade2.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 1)
+        cascade2.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 3)
+
+        cascade3 = experiment.treatment_groups.create!(description: "cascade3")
+        cascade3.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 3)
+        cascade3.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 7)
+      end
+    end
+
+    transaction do
+      Experimentation::Experiment.stale_patients.create!(
+        name: STALE_PATIENTS_EXPERIMENT,
+        start_time: START_TIME,
+        end_time: END_TIME,
+        max_patients_per_day: PATIENTS_PER_DAY,
+        filters: FILTERS
+      ).tap do |experiment|
+        _control_group = experiment.treatment_groups.create!(description: "control")
+
+        single_group = experiment.treatment_groups.create!(description: "single_notification")
+        single_group.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 0)
+
+        cascade1 = experiment.treatment_groups.create!(description: "cascade1")
+        cascade1.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 0)
+        cascade1.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 3)
+        # Stale (5 buckets): control, 0,  0/3, 0/7, and 0/3/7
+
+        cascade2 = experiment.treatment_groups.create!(description: "cascade2")
+        cascade2.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 0)
+        cascade2.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 7)
+
+        cascade3 = experiment.treatment_groups.create!(description: "cascade3")
+        cascade3.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 0)
+        cascade3.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 3)
+        cascade3.reminder_templates.create!(message: "notifications.set03.basic", remind_on_in_days: 7)
+      end
+    end
+  end
+
+  def down
+    Experimentation::Experiment.current_patients.find_by_name(CURRENT_PATIENTS_EXPERIMENT).cancel
```

Comment:
Down migration is not the inverse of the up migration.

Reply:
I'm assuming that by saying * Down migration is not the inverse of the up migration*, you meant we aren't undoing cancel of 2 experiments, right? Did you mean to say something else?

Down migration will cancel (soft-delete) the created experiments. We cannot (shouldn't?) undo the soft-deletion of previous experiments because that will lead to unintended consequences.